Draft: Add Japanese translation

hilot06 opened the pull request:
This request is to add an Japanese translation.
The translation file is in [src/app/i18n/locale/ja-jp.ts](https://github.com/hilot06/nxapi/blob/i18n/src/app/i18n/locale/ja-jp.ts).

I couldn't translate all because I don't know meaning of the specific words (ex. `NSA` or/and `Coral` ).
If you let me know about meaning, I'll fix it. (... and may mark ready for review)

-----

Thank you for your time to look this PR!
hilot06

P.S. Ah, I made an typo in a commit name...

- Closes #59.

hilot06 commented:
To Japanese Translators,

タイプミスをしていたり、そもそも間違っていたりなどなど色々不備がある可能性があります。
拙作の[日本語ファイル](https://github.com/hilot06/nxapi/blob/i18n/src/app/i18n/locale/ja-jp.ts)と、元々こちらのプロジェクトに入っている[英語ファイル](https://github.com/samuelthomas2774/nxapi/blob/i18n/src/app/i18n/locale/en-gb.ts)を比較頂けると幸いです。
修正点があれば恐れ入りますがCommit頂けると嬉しいです。このPull Requestへのコメントのみでも泣いて喜びます…！
なにとぞよろしくお願いいたします。

samuelthomas2774 reviewed (changes requested):
Thank you for adding this! I've added comments for some keys that need checking; [I've also added some extra keys for things I missed since this was opened](https://github.com/samuelthomas2774/nxapi/compare/b9b9f6a8d74d1ba24ef8b84fc502d04985e64ed7..f8ab3721714849a77b9feb056aa291bbacd91019#diff-6439e40dd9e95cc1adf4effe25ea70a75c5a52dc0e04d831b9416f1a50da8d54).

> I couldn't translate all because I don't know meaning of the specific words (ex. NSA or/and Coral ).

NSA (Network Service Account), Coral, Moon, etc. are internal terms used by Nintendo, these don't need to be translated.

samuelthomas2774 commented on src/app/i18n/index.ts:
```diff
@@ -27,6 +27,13 @@ export const languages = {
         ],
     },
+    'ja-JP': {
+        name: 'Japanese',
```
The name here is shown in the about this app window using `app:translation_credits` (e.g. using the non-localised name in #66:), so this should also be localised.

<img width="396" alt="Screenshot 2023-06-02 at 23 24 48" src="https://github.com/samuelthomas2774/nxapi/assets/8474065/abb22423-9c69-4af4-989d-ebee62436bbf">

(Thanks @Sarayalth for pointing this out.)

hilot06 commented on src/app/i18n/locale/ja-jp.ts:
```diff
@@ -0,0 +1,352 @@
+
+    licence: LICENCE_NOTICE,
+    credits: CREDITS_NOTICE,
+    translation_credits: 'Japanese translation by hilot06.',
```
OK. I'll change it right away.

hilot06 commented on src/app/i18n/locale/ja-jp.ts:
```diff
@@ -0,0 +1,352 @@
+
+    startup: {
+        heading: 'スタートアップ',
+        login: 'Windowsにログインした時に開く',
```
Ah, OK. So I'll change it to 'OS'.

hilot06 replied:
Ah OK. But I think users will confuse because they'll think about NSA.
So I'll use 'OS' instead.

hilot06 commented on src/app/i18n/locale/ja-jp.ts:
```diff
@@ -0,0 +1,352 @@
+        minutes_one: '{{count}}分',
+        minutes_other: '{{count}}分',
+
+        presence_inactive: 'オンライン (ゲーム未起動)',
```
I translated "Offline (concole online)" to "Online (no game has launched)".

hilot06 commented:
Thanks a lot to Samuel and Sarah!
I've changed some bad translations and added `na_auth`. Please checked it out.
Also I've checked all translations again, and fix some translations.

hilot06 commented:
i knew i can request a review 20 minutes ago...
it's difficult to operate github...

nennneko5787 commented:
I have revised the text to fit this document, what do you think?
https://www.jtf.jp/pdf/jtf_style_guide.pdf

https://github.com/nennneko5787/nxapi/blob/i18n/src/app/i18n/locale/ja-jp.ts
